SharperUniverse.Tests/PersistenceTests.cs

```diff
@@ -36,13 +40,9 @@ public async Task CanSaveWithoutDuplicates()
 			await barSystem.RegisterComponentAsync(barComponent);
 
 			var provider =
-				new LiteDbProvider(new List<ISharperSystem> {fooSystem, barSystem}, runner)
-				{
-					ConnectionString = $"{Path.GetTempPath()}/SU.db"
-				};
-
+				new LiteDbProvider(new List<ISharperSystem> {fooSystem, barSystem}, runner);
 
-			var id = provider.Save(new List<BaseSharperComponent>
+			var id = await provider.Save(new List<BaseSharperComponent>
```

Comment:
Just a reminder, anything that returns a `Task` or `Task<T>` that's in an interface should be suffixed with `Async`.